ci_pod/nginx/ibm_conf.conf

```diff
@@ -0,0 +1,221 @@
+server {
+  ## elasticsearch by ip with basic auth, with wrong port 9201 
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+        ssl_certificate /ibm_crt.crt;
+        ssl_certificate_key /ibm_private.key;
+
+	listen 443 ssl default_server;
+
+	root /var/www/html;
+
+	# Add index.php to the list if you are using PHP
+	index index.html index.htm index.nginx-debian.html;
+
+	server_name _;
+
+	location / {
+
+		auth_basic "Restricted Content";
+        auth_basic_user_file /etc/nginx/.htpasswd;
+		proxy_pass $INTERNAL_URL;
+   
+	}
+}
+
+server {
+        # elasticsearch by hostname
+        listen 80;
+        listen [::]:80;
+
+        # SSL configuration
+        #
+        ssl_certificate /ibm_crt.crt;
+        ssl_certificate_key /ibm_private.key;
+
+        listen 443 ssl;
+
+        root /var/www/html;
+
+        # Add index.php to the list if you are using PHP
+        index index.html index.htm index.nginx-debian.html;
```

Comment:
Again, be as specific as possible even if it's a bit more verbose.  This makes it harder for someone to misunderstand.
```
        # Uncomment if you are using php
        # index index.html index.php index.htm index.nginx-debian.html;
        # Comment out if you are using php
        index index.html index.htm index.nginx-debian.html;

```